zipkin-lens/src/translations/fr/messages.json

```diff
@@ -0,0 +1,36 @@
+{
+  "1 Result": "1 Résultat",
+  "Address": "Adresse",
+  "Annotations": "Annotations",
+  "Archive Trace": "Trace Archivée",
+  "Dependencies": "Dépendances",
+  "Depth": "Profondeur",
+  "Download JSON": "Télécharger JSON",
+  "Duration": "Durée",
+  "End Time": "Fin",
+  "Find a trace": "Trouver une trace",
+  "Limit": "Limite",
+  "None": "Aucun",
+  "Parent ID": "ID Parent",
+  "Please select criteria in the search bar. Then, click the search button.": "S'il vous plaît sélectionner un critère dans la barre de recherche. Et, cliquer sur le bouton de recherche",
+  "Please select the start and end time. Then, click the search button.": "S'il vous plaît sélectionner la date de début et de fin. Et, cliquer sur le bouton de recherche",
+  "Relative Time": "Temps Relatif",
+  "Root": "Racine",
+  "Search Dependencies": "Rechercher des dépendances",
+  "Search Traces": "Rechercher des traces",
+  "Searching has been disabled via the searchEnabled property. You can still view specific traces of which you know the trace id by entering it in the \"Trace ID...\" textbox on the top-right.": "La recherche a été désactivée via la propriété searchEnabled. Vous pouvez toujours afficher des traces spécifiques dont vous connaissez l'identifiant de trace en le saisissant dans la zone de texte \"ID Trace ... \" en haut à droite.",
+  "Services": "Services",
+  "Span ID": "ID Section",
```

Comment:
Should the span word be translated ? I think section (I've actually never find a good translation).

Reply:
I agree with you, I would like to conserve span too but I want to translate all terms... 
I saw Spanish translation who use "section" and I use the same idea.